devops/scripts/create_esrp_job.ps1

```diff
@@ -0,0 +1,87 @@
+# create_esrp_job.ps1
```

Comment:
Remove this line. In this project we do not comment current file name as first line in the respective file.